packages/utils/telemetry-utils/api-report/telemetry-utils.api.md

```diff
@@ -396,7 +396,10 @@ export enum TelemetryDataTag {
 export type TelemetryEventCategory = "generic" | "error" | "performance";
 
 // @public
-export type TelemetryEventPropertyTypeExt = string | number | boolean | undefined | (string | number | boolean)[] | {
+export type TelemetryEventPropertyTypeExt = string | number | boolean | undefined | (string | number | boolean | undefined | {
```

Comment:
We should do this change in a separate PR to highlight and isolate it from the rest of this change.

Reply:
And what I would say in _that_ PR is if we're doing this we might as well go full-on `Jsonable`.